apps/meteor/app/lib/server/functions/saveUser.js

```diff
@@ -416,8 +416,9 @@ export const saveUser = async function (userId, userData) {
 		updateUser.$set.language = userData.language;
 	}
 
-	if (typeof userData.requirePasswordChange !== 'undefined') {
-		updateUser.$set.requirePasswordChange = userData.requirePasswordChange;
+	updateUser.$set.requirePasswordChange = userData?.requirePasswordChange || false;
```

Comment:
This will cause the flag to be turned off anytime the user is saved unless it is explicitly turning the flag on. Is that really the intended behavior?

Reply:
As far as I checked, everytime the user is saved the `requirePasswordChange` is sent (along with some other fields), so that wouldn't be a problem. But if you worry about unintended change, we can keep as it was and just add a validation to delete the `requirePasswordChangeReason` if the `requirePasswordChange` is set to false.

Reply:
That would be the best approach, thanks!